src/containers/Error/index.js

```diff
@@ -1,78 +1,25 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import ReactTable from 'react-table';
-
-class ErrorComponent extends React.Component {
-  static buildMessage(d, key) {
-    let result = 'Unknown';
-
-    if (d.fileLocation) {
-      result = '';
-      if (d.fileLocation[key].line) {
-        result += `Line: ${d.fileLocation[key].line}`;
-      }
-
-      if (d.fileLocation[key].column) {
-        result += ` Col: ${d.fileLocation[key].column}`;
-      }
-    }
-
-    return result;
-  }
-
-  static buildStartLocation(d) {
-    return ErrorComponent.buildMessage(d, 'start');
-  }
-
-  static buildEndLocation(d) {
-    return ErrorComponent.buildMessage(d, 'end');
-  }
-
-  render() {
-    const columns = [{
-      Header: 'Type',
-      accessor: 'type',
-      width: 100,
-    },
-    {
-      Header: 'Name',
-      accessor: 'name',
-      width: 200,
-    },
-    {
-      Header: 'File',
-      accessor: 'fileName',
-      width: 100,
-    },
-    {
-      id: 'startLocation',
-      Header: 'Start Location',
-      accessor: d => ErrorComponent.buildStartLocation(d),
-      width: 100,
-    },
-    {
-      id: 'endLocation',
-      Header: 'End Location',
-      accessor: d => ErrorComponent.buildEndLocation(d),
-      width: 100,
-    },
-    {
-      Header: 'Message',
-      accessor: 'shortMessage',
-      width: 600,
-    },
-    ];
-
-    return (<ReactTable
-          data={this.props.errors}
-          pageSize={5}
-          columns={columns}
-        />);
-  }
-
-  static propTypes = {
-    errors: PropTypes.array.isRequired,
-  };
-}
-
-export default ErrorComponent;
+import styled from 'styled-components';
+import { connect } from 'react-redux';
+import { ErrorLogger } from '@accordproject/cicero-ui';
+
+const ErrorWrapper = styled.div`
+  bottom: 0;
+  position: absolute;
+  width: 100%;
+`;
+const ErrorContainer = props => (
+  <ErrorWrapper>
+    <ErrorLogger errors={props.errors}/>
+  </ErrorWrapper>
+);
+
+ErrorContainer.propTypes = {
+  errors: PropTypes.array.isRequired,
+};
+const mapStateToProps = state => ({
+  errors: state.modelState.error ? [state.modelState.error] : [],
```

Comment:
I'm not sure I understand this part. Is this ternary necessary when this is plugged into the store?

Reply:
this will be refactored once we have the logic and grammar components also sending errors to the store. i think here we will be concatenating all of the errors before mapping them to props (or refactoring the error component to receive each type of error as a separate prop).

Reply:
Ah I see, makes sense!